.github/workflows/run_linter_yaml.yaml

```diff
@@ -0,0 +1,32 @@
+# Author: James Garriss
+# Purpose: Run the linter against YAML files
+
+name: Lint Check for YAML
+
+# Run whenever YAML files are checked in
+on:
+  push:
+    paths:
+      - '**.yml'
+      - '**.yaml'
+  workflow_dispatch:
+
+jobs:
+  yaml-lint-check:
+    name: MegaLint YAML Check
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Setup Config File
+        run: cp Testing/Linting/MegaLinter/.mega-linter-yaml.yml .mega-linter.yml
+        if: always()
```

Comment:
Not sure this is being used correctly.  This causes the step to always execute  and return true (even if cancelled). 
Consider something like:
`if: ${{ !cancelled() }}`

Reply:
Those are running as intended, or rather they were.  They were are holdover from when both linters were run in the same workflow.  As they are no longer necessary, removed.